plugins/main/tap/tap_test.go

```diff
@@ -0,0 +1,331 @@
+// Copyright 2022 Arista Networks
```

Comment:
could you please double check that it is correct copyright or not.

Reply:
Changed to: 2022 CNI authors
Looking at the other files, looks like this is correct